### ISSUE TITLE
revisit publishing workflows for content types

### ISSUE THREAD

**Issue by jamestranovich-noaa** [weather-gov/weather.gov#2022]
### Description

We have removed the requirement to preview a change before publishing, since that was really only for weather stories and we are not using weather stories anymore.

Nonetheless, we may want to implement custom workflows in the future. (Note that for advanced usage, Drupal has a well known `workflow` [module](https://www.drupal.org/project/workflow)). 

#### Current Behavior

Currently we are using the default Drupal behavior for publication, which optionally allows you to preview: 

![image](https://github.com/user-attachments/assets/76ad2c87-3a3d-44af-bf62-fc68b4d019a2)

You can set the status to `Published`, `Draft`, or `Archived`.

In addition, the sidebar has various settings for metadata attached to the update: 

![image](https://github.com/user-attachments/assets/4903dfd1-f84d-47d2-b656-2d8a0a06e36a)

Each subsequent update will create a revision automatically and these can be reverted: 

![image](https://github.com/user-attachments/assets/a09258f3-89a1-43c8-a7d4-da51319ddf8f)

I believe this should be enough for our content type modification needs but in the future we may want to customize this for a per user workflow or publishing experience (for the newer content types). 

#### Future behavior

Questions to discuss/answer (some are contingent on research/design findings): 

- [ ] Who are our users and what do they need? Currently they are forecasters who create and maintain evergreen content about their offices and pages.
- [ ] Given the above, do we need custom workflows for other content types?
- [ ] What are the pain points with the current CMS when it comes to publishing? Are there any extant process from that CMS that we should adapt or revise?
- [ ] How do we address the corner cases here? E.g., saving changes to a published content type that shouldn't be published yet?

### Acceptance criteria

- [ ] Design/engineering discusses the default publishing behavior and determines if any extra workflow is needed (beyond default Drupal behavior)

**Comment by colinmurphy01** [comment]
Curious to get a sense of size/effort of this?

**Comment by jamestranovich-noaa** [comment]
Well, for now this is a placeholder so we can discuss at the Design/Eng cowork meeting tomorrow (October 31st). Hopefully I'll have more detail on scope afterwards.

**Comment by jamestranovich-noaa** [comment]
Oof, we did not meet today. So we'll discuss next week.

**Comment by kmranjo** [comment]
No, I didn't think this one was assigned for last sprint, was it? I did raise the workflow question with design since we had collaborated on the original version, so we could probably put together a new one fairly quickly.
